src/libecalc/common/utils/rates.py

```diff
@@ -160,6 +160,12 @@ def convert_none_to_nan(cls, v: float, field: ModelField) -> TimeSeriesValue:
     def __len__(self) -> int:
         return len(self.values)
 
+    def __lt__(self, other) -> bool:
+        if not isinstance(other, TimeSeries):
+            return NotImplemented
```

Comment:
Should add a comment and reference why NotImplemented is used here instead of raising exception imo